### PR TITLE
added led control file for mrvll platform. And corrected fancontrol file

### PR DESCRIPTION
**Pull request by shilimkarvg**
**- What I did**
added led control file for mrvll platform. And corrected fancontrol file
**- How I did it**
tested on marvell hardware. 
**- How to verify it**
execute with sensors command
**- Description for the changelog**

### CODE REVIEW

File: device/marvell/armhf-marvell_et6448m_52x-r0/led_control.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    from sonic_led.led_control_base import LedControlBase
+except ImportError, e:
+    raise ImportError(str(e) + " - required module not found")
+
+
+
+class LedControl(LedControlBase):
+    """Platform specific LED control class"""
+    def port_link_state_change(self, portname, state):
+        return
```
Comment (jleveque) [inline device/marvell/armhf-marvell_et6448m_52x-r0/led_control.py]: If you are not using ledd to control the LEDs on this platform, you can use the new mechanism to disable the daemon altogether. See this PR https://github.com/Azure/sonic-buildimage/pull/3920 for an example.

Comment (shilimkarvg) [reply device/marvell/armhf-marvell_et6448m_52x-r0/led_control.py]: @jleveque comment addressed

**Comment by jleveque** [comment]
Please fix conflicts

**Comment by shilimkarvg** [comment]
@lguohan @jleveque  do u have any other comments.

**Pull request merged**